src/pages/elements/NewNotification.js

```diff
@@ -0,0 +1,520 @@
+import React, { Fragment } from 'react';
+import { Query } from 'react-apollo';
+import ApolloClient from 'apollo-boost';
+import gql from 'graphql-tag';
+
+import "@patternfly/react-core/dist/styles/base.css";
+
+import { Link } from 'react-router-dom';
+import makeAnimated from 'react-select/animated';
+import CreatableSelect from 'react-select/creatable';
+import Page from '../../components/Page';
+import { Select, SelectService, SelectRecepient, Option, MultiValue } from "../../components/Select";
+import { sortByLabel, sortByValue, ColourStyles} from '../../components/Utils';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  TextArea,
+  FormSelect,
+  FormSelectOption,
+  Button,
+  Checkbox
+} from '@patternfly/react-core';
+
+const client = new ApolloClient({
+  uri: window.GRAPHQL_URI,
+});
+
+const axios = require('axios');
```

Comment:
Instead of using axios, can't we use `Query` from `react-apollo`?
https://www.apollographql.com/docs/react/data/queries/

I'm uneasy about having the whole project using that component and switching to `axios` in this particular component.

Reply:
I'm skipping reviewing this file for now, as I'd like to see if we can change the approach here.

Reply:
We use const `client = new ApolloClient(...)` and `client.query(...)` like the doc https://www.apollographql.com/docs/react/get-started to do graphql queries. `axios` is only used for talking to qontract-api.